theano/sandbox/tests/test_weighted_select.py

```diff
@@ -0,0 +1,70 @@
+import numpy
+from theano import config, function, tensor
+from theano.sandbox import multinomial
+
+def test_select_distinct():
+    p = tensor.fmatrix()
```

Comment:
Please add a quick comment at the beginning of each test simply to state what behavior the test seeks to validate.